src/content/code/plugin-api/camera/camera-aspect-ratio.ts

```
@@ -0,0 +1,21 @@
+
+// Example 1: Access and log the camera's current aspect ratio
+const cameraAspectRatio = reearth.camera.aspectRatio;
+
+if (cameraAspectRatio !== undefined) {
+  console.log("Current Camera Aspect Ratio:", cameraAspectRatio);
+} else {
+  console.log("Camera aspect ratio is undefined.");
+}
+
+// Example 2: Adjust camera view based on aspect ratio
+const cameraAspectRatio = reearth.camera.aspectRatio;
```

Biome

[error] Shouldn't redeclare 'cameraAspectRatio'. Consider to delete it or rename it.

'cameraAspectRatio' is defined here:


(lint/suspicious/noRedeclare)